src/schemas/json/uplugin.json

```diff
@@ -0,0 +1,401 @@
+      "type": "boolean",
+      "default": false
+    },
+    "HasExplicitPlatforms": {
+      "description": "When true, an empty SupportedTargetPlatforms is interpreted as 'no platforms' with the expectation that explicit platforms will be added in plugin platform extensions",
+      "type": "boolean",
+      "default": false
+    },
+    "Installed": {
+      "description": "Signifies that the plugin was installed on top of the engine",
+      "type": "boolean",
+      "default": false
+    },
+    "IsBetaVersion": {
+      "description": "Marks the plugin as beta in the UI",
+      "type": "boolean",
+      "default": false
+    },
+    "IsExperimentalVersion": {
+      "description": "Marks the plugin as experimental in the UI",
+      "type": "boolean",
+      "default": false
+    },
+    "IsHidden": {
+      "description": "For auto-generated plugins that should not be listed in the plugin browser for users to disable freely.",
+      "type": "boolean",
+      "default": false
+    },
+    "IsPluginExtension": {
+      "description": "If true, this plugin from a platform extension extending another plugin",
+      "type": "boolean",
+      "default": false
+    },
+    "RequiresBuildPlatform": {
+      "description": "For plugins that are under a platform folder (eg. /PS4/), determines whether compiling the plugin requires the build platform and/or SDK to be available",
+      "type": "boolean",
+      "default": false
+    },
+    "Category": {
+      "description": "The name of the category this plugin",
+      "type": "string",
+      "default": "Other"
+    },
+    "CreatedBy": {
+      "description": "The company or individual who created this plugin.",
+      "type": "string"
+    },
+    "CreatedByURL": {
+      "description": "Hyperlink URL string for the company or individual who created this plugin. This is optional.",
+      "type": "string"
+    },
+    "Description": {
+      "description": "Description of the plugin",
+      "type": "string"
+    },
+    "DocsURL": {
+      "description": "Documentation URL string.",
+      "type": "string"
+    },
+    "EditorCustomVirtualPath": {
+      "description": "Optional custom virtual path to display in editor to better organize.",
+      "type": "string"
+    },
+    "EnabledByDefault": {
+      "description": "Whether this plugin should be enabled by default for all projects",
+      "type": "string",
+      "enum": ["Unspecified", "Enabled", "Disabled"]
+    },
+    "EngineVersion": {
+      "description": "Version of the engine that this plugin is compatible with",
+      "type": "string"
+    },
+    "FileVersion": {
+      "description": "Descriptor version number.",
+      "type": "number",
+      "default": 3
+    },
+    "FriendlyName": {
+      "description": "Friendly name of the plugin",
+      "type": "string"
+    },
+    "LocalizationTargets": {
+      "description": "List of all localization targets associated with this plugin",
+      "type": "array",
```

Comment:
Optional fix: add `"uniqueItems": true` to make sure that there are no duplicates.

Reply:
Optional fix implemented in 262cb20dcfc106d02304efb65aa070fa0db880cc.